### PR TITLE
Update version number on 1.2.0 branch

### PR DESCRIPTION
**Pull request by geovations**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -16,7 +16,7 @@ endif()
 
 # gbdxm version
 set(gbdxm_VERSION_MAJOR 1)
-set(gbdxm_VERSION_MINOR 1)
+set(gbdxm_VERSION_MINOR 22)
```
Comment (rddesmond) [inline CMakeLists.txt]: Fix typo?

**Review by rddesmond — CHANGES_REQUESTED** [review]

**Review by rddesmond — APPROVED** [review]

**Comment by geovations** [comment]
test this please

**Comment by geovations** [comment]
test this please

**Comment by geovations** [comment]
test this please

**Comment by geovations** [comment]
test this please

**Comment by geovations** [comment]
test this please

**Comment by geovations** [comment]
test this please

**Pull request merged**